include/mbedtls/cipher_internal.h

```diff
@@ -114,6 +118,17 @@ typedef struct
     const mbedtls_cipher_info_t *info;
 } mbedtls_cipher_definition_t;
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+typedef struct
+{
+    psa_key_slot_t slot;
+    unsigned char slot_state; /*!< 0: The slot is unset.
```

Comment:
Since these numbers are used onwards, why won't we define them? 
PSA_CONTEXT_UNSET, PSA_CONTEXT_OWNED, PSA_CONTEXT_UNOWNED, or something like that.

Reply:
Good idea.